endpoints/openrtb2/video_auction.go

```diff
@@ -472,12 +472,7 @@ func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.Bi
 	}
 
 	if &videoRequest.User != nil {
-		bidRequest.User = &openrtb.User{
-			BuyerUID: videoRequest.User.Buyeruids["appnexus"], //TODO: map to string merging
-			Yob:      videoRequest.User.Yob,
-			Gender:   videoRequest.User.Gender,
-			Keywords: videoRequest.User.Keywords,
-		}
+		bidRequest.User = videoRequest.User
```

Comment:
Is the `nil` check necessary since it's now just wrapping an assignment?

Reply:
Removed nil check, it was not needed